docs/graphviz.rst

```diff
@@ -123,10 +123,60 @@ example above.
 The reStructuredText will be parsed, and after resolving any cross references,
 will be substituted back into the graph definition as follows:
 
-- A :graphvizattr:`label` will be generated from the text content.
-- If there is at least one ``reference`` node, only the last such node is
+- A :graphvizattr:`label` will be generated from the cross reference's text content.
+- If there is at least one cross referable target, only the last such target is
   considered, and:
 
   - an :graphvizattr:`href` will be generated from its URL.
   - a :graphvizattr:`tooltip` will be generated from its title/tooltip, if any.
   - a :graphvizattr:`target` will be generated from its target, if any.
+
+Overriding the cross-reference label
+************************************
+
+The :graphvizattr:`label` generated with the new ``xref`` attribute uses escaped HTML characters.
```

Comment:
generated from the new ``xref`` attribute uses the graphviz HTML-like attribute syntax which is not compatible with record-based nodes.  For record-based nodes, or in other cases where it is desired to override the label manually, simply specify the label after the xref attribute.

Reply:
I might take a little liberty with this wording because
> uses the graphviz HTML-like attribute

doesn't imply that the text content within angle brackets has escaped HTML characters. Even though HTML-like labels very much support HTML characters.

I understand where you're going with this (there are a lot of recommended use cases for HTML-like labels), but the whole point of demonstrating overridden labels was because of the escaped HTML in the generated label.

Otherwise, a clever person might try the following without knowing about the escaped HTML:
```rst
.. graphviz::
    digraph {
        class [ shape=record ]
        record_node [
             xref=":ref:`\<f0\> field 1 \| \<f1> field 2 <target>`" 
             shape=record
        ]
        // notice the edge point to a specific field
        class -> record_node:f0
    }
```
Which results in a node that looks like 
![image](https://user-images.githubusercontent.com/14963867/198750466-e740d1c2-abba-49aa-82a4-9c0c63107fde.png)
but the edge can't locate the "f0 port" because the HTML was escaped escaped in the xref's label, resulting in a 
>Warning: node record_node, port f0 unrecognized

Reply:
I see --- I guess you are saying that it is partially incorrect to say that the HTML-like attribute syntax is incompatible with record-based nodes --- because in your example the "|" is getting parsed correctly.

I suppose instead we can say:

generated from the new xref attribute cannot be used to specify record-based nodes, because angle brackets (``<`` and ``>``) in the fragment's text content will be escaped such that graphviz interprets them as literal text rather than specifying record fields.  For record-based nodes, or in other cases where it is desired to override the label manually, simply specify the label after the xref attribute.

Reply:
Yes, exactly. Also, pointing directly at record nodes might be too narrow for the thesis of this section. I use record nodes in the example, so that can be implied as a use case that requires overriding the label. But HTML-like syntax is useful in a number of situations, not just record nodes. I'm having trouble picturing other cases where a user would need to have HTML-like label syntax and an xref attribute, but I'm sure someone will.

I'm currently working with this wording:
>The [label attribute](https://graphviz.org/docs/attrs/label/) generated with the new xref attribute uses graphviz’ HTML-like attribute syntax, but any HTML characters found within the xref parsed fragments’ text are escaped (eg. `< f0 >` becomes `&lt; f0 &gt;`). In some graphs, it is preferable to use unescaped HTML characters in the [label attribute](https://graphviz.org/docs/attrs/label/). In this case, the generated [label attribute](https://graphviz.org/docs/attrs/label/) can be overridden by manually specifying the [label attribute](https://graphviz.org/docs/attrs/label/) after the xref attribute.

Reply:
Okay.